### DISCUSSION TITLE
google search interstitial and system webview banking app problem.

### DISCUSSION THREAD

**Discussion by realpinman** [bromite/bromite#1395] [General]
Hi, I'm seeing some strange behaviour after replacing chrome browser and system webview on my android custom ROM (Pixel extended).

Whenever i try a search (with google set as search engine) i get an interstitial page "click here if your browser does not redirect you" and also the search result page i then land on seems not to be the full google page but a knocked down version.

Also i have replaced the system webview with the bromite webview this this now broken my revolut banking app i use (which has webview elements when topping up).

Would the extra privacy/ad blocking features be casuing this? any fix? thanks.

**Comment by Oymate** [comment]
Screenshot/record?
